client/landing/stepper/declarative-flow/internals/hooks/use-migration-experiment/index.ts

```diff
@@ -0,0 +1,9 @@
+import { useExperiment } from 'calypso/lib/explat';
+
+export function useMigrationExperiment( flowName: string ) {
+	const [ , experimentAssignment ] = useExperiment(
+		'calypso_signup_onboarding_site_migration_flow_202501_v1'
+	);
+
+	return 'treatment' === experimentAssignment?.variationName && 'site-migration' === flowName;
```

Comment:
Should it be migration instead of site-migration?
```suggestion
	return 'treatment' === experimentAssignment?.variationName && 'migration' === flowName;
```

Reply:
It depends on which flows we intend to test.

The `site-migration` flow uses the adjusted steps, the `migration` flow does not use any.

I looked at the code and the `migration-signup` flow *does* include two of the four adjusted steps (identify and upgrade). But do we intend to test that flow, too?

Reply:
We chatted about it more in Slack and agreed `site-migration` is the flow we're targeting: p1736368676222639-slack-C0Q664T29

Reply:
Yes, I was misunderstanding the goal,  sorry 🤦